src/ab-test-preview.js

```diff
@@ -0,0 +1,36 @@
+import { getLetter } from '../src/utils';
+
+document.querySelectorAll( '.ab-test-xb-preview' ).forEach( function ( xb ) {
+
+	let templates = document.querySelectorAll( 'template[data-parent-id="' + xb.dataset.clientId + '"]' );
+	let tabContainer = document.querySelector( '.ab-test-xb-preview__tabs[data-client-tabs="' + xb.dataset.clientId + '"]' );
+	const tabContent = xb.querySelector( '.ab-test-xb-preview__content' );
+
+	for ( let i = 0; i < templates.length; i++ ) {
+
+		const tab = document.createElement( 'button' );
+
+		if ( i === 0 ){
+			const variant = templates[i].content.cloneNode( true );
+			tab.className = 'ab-test-xb-preview__tab active';
+			tabContent.innerHTML = '';
+			tabContent.appendChild( variant );
```

Comment:
I had made some changes to this section so Im not entirely sure if it will still be relevant. But I currently have the first tab being active. I have an issue when setting a preview for the second tab, then the first tab is also active.

Reply:
Still relevant but will need to handle getting the right tab item to fire the click event on rather than just using the first one.

Reply:
This should have been resolved in a newer commit